tests/k8s_utils.go

```diff
@@ -199,20 +215,26 @@ func (k *KubeClient) waitForPVCBound(pvc, ns string) (corev1.PersistentVolumeCla
 		if o.Status.Phase == corev1.ClaimBound {
 			return o.Status.Phase, nil
 		}
+		fmt.Printf("waiting for PVC {%s} in namespace {%s} to get into bound state\n", pvc, ns)
 		time.Sleep(5 * time.Second)
 	}
 }
 
-// createPVC creates the given PVC and ensure that PVC bound
-func (k *KubeClient) createPVC(pvc *corev1.PersistentVolumeClaim) error {
+// createPVC will create PVC and wait till PVC gets into bound state if shouldWaitForProvision
+// is set to true else createPVC will return immediately after creating PVC
+func (k *KubeClient) createPVC(pvc *corev1.PersistentVolumeClaim, shouldWaitForProvision bool) error {
 	_, err := k.CoreV1().PersistentVolumeClaims(pvc.Namespace).Create(pvc)
 	if err != nil {
 		if !k8serrors.IsAlreadyExists(err) {
 			return err
 		}
 	}
-	_, err = k.waitForPVCBound(pvc.Name, pvc.Namespace)
-	return err
+
+	if shouldWaitForProvision {
```

Comment:
since we have a different function to wait for PVC bound, let's remove this from `createPVC` function.

Reply:
Update in all respective places